SMLHelper/Initializer.cs

```diff
@@ -20,12 +20,14 @@ public class Initializer
         [Obsolete("This method is for use only by QModManager.", true)]
         public static void PrePatch()
         {
-            Logger.Initialize();
+            Logger.Initialize(); 
 #if SUBNAUTICA
             Logger.Log($"Loading v{Assembly.GetExecutingAssembly().GetName().Version} for Subnautica", LogLevel.Info);
 #elif BELOWZERO
             Logger.Log($"Loading v{Assembly.GetExecutingAssembly().GetName().Version} for BelowZero", LogLevel.Info);
 #endif
+            TechTypePatcher.cacheManager.LoadCache();
+            CraftTreeTypePatcher.cacheManager.LoadCache();
```

Comment:
Add a Debug level log before these new lines indicating that cache files are being loaded

Reply:
Will do